example/scalalib/linting/1-scalafmt/src/Foo.scala

```diff
@@ -1,9 +1,4 @@
 package foo
-object Foo{
-def main(args:
-Array[String
-]
-):Unit=
-{println("hello world")
-}
+object Foo {
```

Comment:
This file is meant to be mis-formatted, in order to exercise the Scalafmt workflow as part of the test suite. You should be able to skip the file specifically in `project.excludePaths` I think

Reply:
Oh, good catch. Fixed.